### PR TITLE
Importer: Treat unimportable and not exists as skipped

### PR DESCRIPTION
**Pull request by MartinPetkov**

### CODE REVIEW

File: internal/tfimport/tfimport.go [hunk 1]
```diff
@@ -761,8 +761,10 @@ func planAndImport(rn, importRn runner.Runner, runArgs *RunArgs, skipped map[str
 		// err will be `exit code 1` even when it failed because the resource is not importable or already exists.
 		case NotImportable(output):
 			log.Printf("Import not supported by provider for resource %q\n", cc.Address)
+			skipped[cc.Address] = true
```
Comment (xingao267) [inline internal/tfimport/tfimport.go:764]: With this change, these resources will be listed under `Skipped` instead of `Failed to import`?

Comment (MartinPetkov) [reply internal/tfimport/tfimport.go:764]: Currently they're not listed at all.

**Review by xingao267 — APPROVED** [review]

**Comment by xingao267** [comment]
Do you want to cut a release and use it in the solution templates?

**Comment by MartinPetkov** [comment]
> Do you want to cut a release and use it in the solution templates?

No, not needed right now, it's just something I observed and wanted to update.

**Pull request merged**